documentation/config_file_puppetserver.markdown

```diff
@@ -73,6 +73,19 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 * The `profiler` settings configure profiling:
 
     * `enabled`: If this is set to `true`, Puppet Server enables profiling for the Puppet Ruby code. The default is `false`.
+    
+* The `product` settings configure update checking and analytics data collection:
```

Comment:
I'm fine with breaking this out if it's an option, especially if we're not releasing today. It's just been a surprise that a separate `product.conf` file _is_ an option.

To answer question 2, a separate config file is going to bring the docs folks into it regardless. A new conf file = a new page in the docs = a new link in the nav = probably new redirects since the docs site version selector will point people to versions where that file and doc don't exist.